<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>671</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="login_button">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_edit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_edit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>200</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="maxLength">
       <number>32764</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="username_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="submit_button">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="asset_number_edit">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="asset_number_labe">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Asset No.</string>
      </property>
     </widget>
     <widget class="QLabel" name="asset_name_label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Asset Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="asset_name_edit">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="assignee_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Assignee</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="assignee_edit">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>120</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QListWidget" name="asset_list">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>256</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>400</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionsign_out"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsign_out">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>sign out</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
